src/targets/cpu/gemm.cpp

```diff
@@ -55,7 +55,12 @@ void migemm_impl(tensor_view<T> cmat,
     visit_mat(amat, [&](const auto& a) {
         visit_mat(bmat, [&](const auto& b) {
             auto c = make_mat(cmat);
-            c      = (a * b) * alpha + beta * c;
+            c      = beta * c;
+
+            if(alpha != 0.0)
+            {
+                c = c + alpha * a * b;
+            }
```

Comment:
If alpha = 0, then wouldn't c = beta * c by default? Can you remove the if condition and make it the default behavior?

Reply:
The previous one line code cannot generate results, so I rewrite it as two lines as: c = beta * c; c = c + alpha*A*B. The reason I added an if condition is a straightforward optimization that if alpha is 0, there is no need to perform the matrix multiplication A*B.